.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -29,9 +32,29 @@ assignees: ''
  - [ ] [DIS] Upload release APK and bundle to internal owncloud instance
  - [ ] [GIT] Merge master branch into stable, in owncloud/android-library
  - [ ] [GIT] Merge master branch into stable, in owncloud/android
- - [ ] [DOC] Update documentation with new stuff (notify rocketchat #documentation-internal in advance!)
+ - [ ] [DOC] Update documentation with new stuff by creating [issue](https://github.com/owncloud/docs-client-android/issues)
+
 
+### BUGS & IMPROVEMENTS
 
 _____
 
-### BUGS & IMPROVEMENTS
+## OEM Release
+
+### TASKS:
+
+- [ ] [DIS] Update release notes in app with the proper content for oem release
+- [ ] [GIT] Create and sign tag `oc-android-X.Y.Z_oem` in HEAD commit of `release_X.Y.Z_oem` branch
```

Comment:
Here it is clearer since we call each number with a different letter 👍

Reply:
yes, it's the same. Congruent with naming in every part.